### PR TITLE
New release

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
### Added

- Add module to classify samples by clonality ([#178](https://github.com/nf-core/crisprseq/pull/178))
- Add DrugZ, a module for chemogenetic interaction ([#168](https://github.com/nf-core/crisprseq/pull/168))
- Add Hitselection, a module for subsetting more likely true positives for KO screen based on the protein protein interaction ([#191](https://github.com/nf-core/crisprseq/pull/191))
- Make the use of gene essentiality module more user friendly and readable in the code ([#194](https://github.com/nf-core/crisprseq/pull/194))

### Fixed

- Fix cutadapt 3' and 5' no such variable found bug ([#187](https://github.com/nf-core/crisprseq/pull/187))
- Fix design matrix bug that introduced dots instead of a hyphen ([#190](https://github.com/nf-core/crisprseq/pull/190))
- Make output of FluteMLE optional as when some pathways produce bugs some channels are then empty ([#190](https://github.com/nf-core/crisprseq/pull/190))
- Fix a typo in crisprcleanr/normalize, when a user inputs a file ([#192](https://github.com/nf-core/crisprseq/pull/192))

### CODE REVIEW

File: bin/BAGEL.py [hunk 1]
```diff
@@ -588,15 +587,15 @@ def calculate_bayes_factors(
     coreEss = []
 
     with open(essential_genes) as fin:
-        skip_header = fin.readline()
+        # skip_header = fin.readline()
```
Comment (mirpedrol) [inline bin/BAGEL.py:590]: should this comment be removed? (same for line 598)

File: docs/output/screening.md
Comment (mirpedrol) [inline docs/output/screening.md:1]: Seeing this reminds me that we should add output docs for the new targeted module `clonality_classifier`

**Review by mirpedrol — APPROVED** [review]
Made a couple of comments, otherwise LGTM 🙂

File: docs/output/screening.md
Comment (LaurenceKuhl) [reply docs/output/screening.md:1]: @mirpedrol can you take care of this ?

File: bin/BAGEL.py (hunk 1 above)
Comment (LaurenceKuhl) [inline bin/BAGEL.py:590]:
```suggestion
```

File: workflows/crisprseq_targeted.nf [hunk 2]
```diff
@@ -685,6 +686,19 @@ workflow CRISPRSEQ_TARGETED {
...
+    if (!params.skip_clonality) {
+        CLONALITY_CLASSIFIER (
+            CIGAR_PARSER.out.indels
+            .join(CIGAR_PARSER.out.edition)
+            .map { [it[0], it[1], it[4]] }
+        )
+        ch_versions = ch_versions.mix(CLONALITY_CLASSIFIER.out.versions.first())
+    }
+
+
```
Comment (JoseEspinosa) [inline workflows/crisprseq_targeted.nf:701]:
```suggestion
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Just small suggestions but nothing really important 🚀 
For me is ready to go once you address @mirpedrol comments

File: docs/output/screening.md
Comment (mirpedrol) [reply docs/output/screening.md:1]: I will, I leave it unresolved just as a reminder to myself and to not merge until it's fixed

**Pull request merged**